dice.js

```diff
@@ -0,0 +1,13 @@
+const rollDice = function (rolling) {
+
+    for (let i = 0; i < rolling; i++) {
+
+        const numberOne = Math.floor(Math.random() * 6) + 1;
+
+        const numberTwo = Math.floor(Math.random() * 6) + 1;
+
+        const sum = numberOne + numberTwo;
+
+        console.log("Dice rolled are " + (i + 1) + ", " + numberOne + ", and " + numberTwo + ". Sum is " + sum);
+    }
+};
```

Comment:
The bonus was asking to get the numbers of dice rolled as an arguments,
- so for example if you called the function like this `rollDice(2)` you should get ` Dice rolled are 1 and 3. Sum is 4` as result.
- and if you called it like this `rollDice(4)` you should get `Dice rolled are 1, 4, 2 and 3. Sum is 10`